cmd/bnsapi/handlers/handlers.go

```diff
@@ -563,7 +564,29 @@ func lastChunk(path string) string {
 }
 
 // DefaultHandler is used to handle the request that no other handler wants.
-type DefaultHandler struct{}
+type DefaultHandler struct {
+	Domain string
+}
+
+var availableEndpointsTempl = template.Must(template.New("").Parse(`
+Available endpoints with query parameter:
+
+{{ .}}/account/accounts/?domainKey=_&ownerKey=_ 
+{{ .}}/account/accounts/{accountKey}
+{{ .}}/account/domains/?admin=_&offset=_
+{{ .}}/blocks/{blockHeight}
+{{ .}}/escrow/escrows/?source=_&destination=_&offset=_
+{{ .}}/gconf/{extensionName}
+{{ .}}/multisig/contracts/?offset=_
+{{ .}}/termdeposit/contracts/?offset=_
+{{ .}}/termdeposit/deposits/?depositor=_&contract=_&contract_id=?_offset=_
+
+Available endpoints w/o parameter:
```

Comment:
`Available endpoints w/o parameter` should be `Available endpoints without query parameters`.